### PR TITLE
Tests | logger unit tests

### PR DESCRIPTION
**Pull request by AlexNDRmac**
Hello!

* Type: unit tests
* Link to issue:

In raising this pull request, I confirm the following:

- [X] I have checked that another pull request for this purpose does not exist
- [X] I wrote some tests for this PR
- [ ] I updated the CHANGELOG

Small description of change:
- Unit tests for `CompilerFormatter`

Thanks

### CODE REVIEW

File: unit-tests/Zephir/Test/Logger/Formatter/CompilerFormatterTest.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            'context' => [
+                'unused-variable-external',
+                [
+                    'type' => 'parameter',
+                    'name' => 'param1',
+                    'const' => 0,
+                    'data-type' => 'variable',
+                    'mandatory' => 0,
+                    'reference' => 0,
+                    'file' => realpath(__DIR__.'/../../../../../test/closures.zep'),
```
Comment (sergeyklay) [inline unit-tests/Zephir/Test/Logger/Formatter/CompilerFormatterTest.php]: Can you please use here something a bit more human friendly

**Review by sergeyklay — CHANGES_REQUESTED** [review]

File: unit-tests/Zephir/Test/Logger/Formatter/CompilerFormatterTest.php (hunk 1 above)
Comment (AlexNDRmac) [reply unit-tests/Zephir/Test/Logger/Formatter/CompilerFormatterTest.php]: Ok :) I'll try to use fixture with as friendly as possible path :)

File: unit-tests/Zephir/Test/Logger/Formatter/CompilerFormatterTest.php [hunk 2]
```diff
@@ -47,8 +47,8 @@ public function getWarningContext(): array
                     'data-type' => 'variable',
                     'mandatory' => 0,
                     'reference' => 0,
-                    'file' => realpath(__DIR__.'/../../../../../test/closures.zep'),
-                    'line' => 20,
+                    'file' => realpath(__DIR__.'/../../../../fixtures/logger/closures.zep'),
```
Comment (sergeyklay) [inline unit-tests/Zephir/Test/Logger/Formatter/CompilerFormatterTest.php]: How about adding something like FIXTURES_PATH:
https://github.com/phalcon/zephir/blob/3d587e9200e19cff439a5028fc1ffe53b69dc463/phpunit.xml.dist#L46

**Review by sergeyklay — CHANGES_REQUESTED** [review]

File: unit-tests/Zephir/Test/Logger/Formatter/CompilerFormatterTest.php (hunk 2 above)
Comment (AlexNDRmac) [reply unit-tests/Zephir/Test/Logger/Formatter/CompilerFormatterTest.php]: Yea, great proposal!

**Review by sergeyklay — APPROVED** [review]

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**